addons/mail/static/src/public/discuss_public_boot.js

```diff
@@ -24,7 +24,7 @@ import {
 } from '@web/legacy/utils';
 import * as legacySession from 'web.session';
 
-owl.Component.env = legacyEnv;
+// owl.Component.env = legacyEnv;
```

Comment:
That doesn't break the public page? What has changed that allows removing it?
It's not well covered by tests.

Reply:
It's probably broken right now I don't think we've gotten around to this yet